index.html

```diff
@@ -1,413 +1,259 @@
-          love to help with it
-        </p>
-        <ul class="form-sections">
+    <section class="skill-section-items">
+    <ul>
+      <li>
+        <div class="languages-dropdown-bar">
+        <p>Langauges<span><img src="./images/dropdown..png" alt="languages"></span></p>
+        <div class="skill-items">
+        <ul>
           <li>
-            <label for="name"></label>
-            <input
-              type="text"
-              id="name"
-              name="user_name"
-              maxlength="30"
-              placeholder="Name "
-              required
-            />
-          </li>
+            <span><img src="./images/JS.png" alt="js-icon">Javascript</span></li>
           <li>
-            <label for="mail"></label>
-            <input
-              type="email"
-              id="mail"
-              name="user_mail"
-              placeholder="Your email "
-              required
-            />
-          </li>
+          <span><img src="./images/HTML.png" alt="html-icon">HTML</span></li>
           <li>
-            <label for="msg"></label>
-            <textarea
-              class="textarea"
-              id="msg"
-              name="user_message"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
+            <span><img src="./images/CSS.png" alt="css-icon">CSS</span></li>
+          </ul>
+        </div>
+        </div>
+        </li>
+        <li>
+          <div class="skill-projects-item">
+          <p>Frameworks<span><img src="./images/arrow.png" alt="frames-icon"></span></p>
+          </div>
+          <hr/>
           </li>
           <li>
-            <button class="but" type="submit">Get In Touch</button>
+          <div class="skill-projects-item">
+          <p>Skills<span><img src="./images/arrow.png" alt="skills-icon"></span></p>
+          </div>
+          <hr/>
           </li>
-        </ul>
-      </form>
+    </ul>
     </section>
-  </body>
-</html>
+    </div>
+  </section>
+  <section>
+    <form class="form" action="https://formspree.io/f/xeqndgga" method="post">
+      <h1>Contact Me</h1>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <ul class="form-sections">
+        <li>
+          <label for="name"></label>
+          <input type="text" id="name" name="user_name" placeholder="Name" maxlength="30" required />
+        </li>
+        <li>
+          <label for="mail"></label>
+          <input type="email" id="mail"  placeholder="Enter Email" name="user_mail" required/>
+        </li>
+        <li>
+          <label for="msg"></label>
+          <textarea class="textarea" id="msg" name="user_message" placeholder="Write your message here" maxlength="500" required></textarea>
+        </li>
+        <li>
+        <button type="submit" class="but">Get In Touch</button>
```

Comment:
- [ ] The buttons on your page buttons doesn't have any interaction, please make sure the colors are changed when the user hover over or clicks on the button

|![Screenshot_20220610_181338](https://user-images.githubusercontent.com/25227022/173118019-9a552c8f-01c2-454b-8417-10787db67a22.png)|![Screenshot_20220610_181349](https://user-images.githubusercontent.com/25227022/173118035-e71c1178-87db-4336-a722-844f2bdf41f6.png)|
|-|-|
|Figma buttons design =>|![173092219-080e5e0a-0542-4a16-be80-9db89c79fda1](https://user-images.githubusercontent.com/25227022/173118111-40391e36-c86b-498b-b912-2960b4e47f38.png)|